packages/vs-button/build/index.js

```diff
@@ -0,0 +1,152 @@
+'use strict';
```

Comment:
Does index.js need to be included now? I had been ignoring it and it was being built on prepublish

Reply:
That gets statically generated.

Reply:
👍